R-package/R/zzz.R

```diff
@@ -16,7 +16,10 @@ NULL
 
 .onLoad <- function(libname, pkgname) {
   # Require methods for older versions of R
-  require(methods)
+  if (! is.element("package:methods", search())) {
```

Comment:
We have `methods` by default after R 3.3, right?

Reply:
"Since S4 is implemented in the methods package, you need to make sure it’s available. This is easy to overlook because while the method package is always available in the search path when you’re working interactively, it’s not automatically loaded by Rscript, the tool often used to run R from the command line." (http://r-pkgs.had.co.nz/namespace.html).  According to this same source, the following is best practice for making this package available.  I will update the PR with this change.

```
#' @import methods
NULL
```

Reply:
I know `methods` may not be loaded automatically and we already import it in DESCRIPTION

https://github.com/apache/incubator-mxnet/blob/master/R-package/DESCRIPTION#L16

I am sorry I still didn't understand why we need to make this change.